.github/workflows/test.yml

```diff
@@ -22,9 +22,7 @@ jobs:
           python-version: ${{ matrix.python-version }}
 
       - name: Install dependencies
-        run: |
-          python -m pip install --upgrade pip
-          python -m pip install tox tox-gh-actions
+        run: python -m pip install tox tox-gh-actions
```

Comment:
Why don't we need to upgrade pip here?

Reply:
See discussion here https://github.com/actions/setup-python/issues/506#issuecomment-1262522726